src/app/core/service/input/shortcut.service.ts

```diff
@@ -6,9 +6,16 @@ import { Observable, Subject } from 'rxjs'
 
 export interface Shortcut {
   accelerator: string
+  ref: any
```

Comment:
i recommend `unknown` over `any` it forces you to cast as type when you get to a type inferrance. feel free to ignore it tho

Reply:
I've seen "any" being used in a number of places, so I'll just stick with it.